samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/SpeechRecognitionSamples.java

```diff
@@ -958,12 +961,17 @@ else if (e.getResult().getReason() == ResultReason.NoMatch) {
                 // Last buffer read from the WAV file is likely to have less bytes
                 pushStream.write(Arrays.copyOfRange(readBuffer, 0, bytesRead));
             }
+
+            // Sleep corresponding time for the uploaded audio chunk, to simulate the natural speaking rate.
+            Thread.sleep(bytesRead / 32);
```

Comment:
looks this is simulation is only for the latency measurement. This will mislead the customer who use file as input.
I suggest comment out the sleep and say something like "To measure the latency in real-time recording scenario, uncomment following line to...)